teps/0055-migrate-podlatency-to-taskprelaunchlatency.md

```diff
@@ -0,0 +1,121 @@
+---
+status: proposed
+title: Migrate podLatency to taskPrelaunchLatency
+creation-date: '2021-01-06'
+last-updated: '2021-03-05'
+authors:
+- '@yaoxiaoqi'
+---
+
+# TEP-0055: Migrate `podLatency` to `taskPrelaunchLatency`
```

Comment:
Perhaps we could use `taskRunPrelaunchLatentcy` ?

Reply:
Sure, it's better than `Task`